adapters/huaweiads/huaweiads.go

```diff
@@ -788,39 +778,54 @@ func convertCountryCode(country string) (out string) {
 	return defaultCountryName
 }
 
+func getCountryCodeFromMCC(MCC string) (out string) {
+	var country_mcc = strings.Split(MCC, "-")[0]
```

Comment:
**Nitpick:** Please rename this variable to `countryMCC` as Go's standard naming convention follows camel case and not snake case